frontend/src/pages/org/crawl-detail.ts

```diff
@@ -604,7 +613,45 @@ export class CrawlDetail extends LiteElement {
               `
             : html`<sl-skeleton class="h-6"></sl-skeleton>`}
         </btrix-desc-list-item>
-        <btrix-desc-list-item label=${msg("Reason")}>
+        <btrix-desc-list-item label=${msg("Duration")}>
+        ${this.crawl
+              ? html`
```

Comment:
re: file size, it looks like the API now returns it in `stats.size`. Pull the latest commit to get the type change, and then you can something like this here:
```ts
<sl-format-bytes
  value=${this.crawl.stats.size}
  display="narrow"
></sl-format-bytes>
```